.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -8,25 +8,27 @@ about: Create a report to help us improve
 
 ## 🐛 Bug Report
 
-A clear and concise description of what the bug is.
+<!-- A clear and concise description of what the bug is. -->
 
 ## To Reproduce
 
 Steps to reproduce the behavior:
 
 ## Expected behavior
 
-A clear and concise description of what you expected to happen.
+<!-- A clear and concise description of what you expected to happen. -->
 
 ## Link to repl or repo (highly encouraged)
 
-Please provide either a [repl.it demo](https://repl.it/languages/jest) or a minimal repository on GitHub.
+<!-- Please provide either a [repl.it demo](https://repl.it/languages/jest) or a minimal repository on GitHub. -->
 
 Issues without a reproduction link are likely to stall.
```

Comment:
this should also be commented out, I guess

Reply:
done